### DISCUSSION TITLE
Allure Report extension for Azure DevOps - Release Pipeline

### DISCUSSION THREAD

**Discussion by Nikicyde** [allure-framework/allure2#2717] [Questions & Support]
Hello!

So I have successfully managed to install Allure extension for Azure DevOps, however it is working correctly only with build pipelines
![image](https://github.com/user-attachments/assets/9209a41f-6b40-40eb-8e76-af77bbe46800)

The problem is that most of the teams are using Release Pipelines instead of Build, but there the tab is not being created with the same extension, even when the task itself is succeeded
![image](https://github.com/user-attachments/assets/77acfff5-afbf-45aa-9e22-daeb90f60ff4)

Anyone knows how could I fix this?

Thank you very much!

**Comment by Rubaka** [comment]
Any updates on this issue? why its not described in the official documentation that its not supported?
